integrationtests-jakarta/pom.xml

```diff
@@ -0,0 +1,293 @@
+                <artifactId>transformer-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>transform</id>
+                        <goals>
+                            <goal>transform</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>transform-test</id>
+                        <goals>
+                            <goal>transform</goal>
+                        </goals>
+                        <configuration>
+                            <transformDirectory>${project.build.testOutputDirectory}</transformDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>enforce-banned-dependencies</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <bannedDependencies>
+                                    <excludes>
+                                        <exclude>javax.validation:*</exclude>
+                                        <exclude>javax.el:*</exclude>
+                                        <exclude>javax.persistence:*</exclude>
+                                        <exclude>javax.inject:*</exclude>
+
+                                        <exclude>jakarta.annotation:*:(,2.0.0)</exclude>
+                                        <exclude>jakarta.validation:*:(,3.0.0)</exclude>
+                                        <exclude>jakarta.el:*:(,4.0.0)</exclude>
+                                        <exclude>jakarta.persistence:*:(,3.0.0)</exclude>
+                                        <exclude>jakarta.inject:*:(,2.0.0)</exclude>
+                                    </excludes>
+                                </bannedDependencies>
+                            </rules>
+                            <fail>true</fail>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <configuration>
+                            <trimStackTrace>false</trimStackTrace>
+                            <useFile>false</useFile>
+                        </configuration>
+                    </execution>
+                </executions>
+                <configuration>
```

Comment:
Just marking it with a comment, but this needs to be removed once we've made the tests work with Jakarta.